src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -702,10 +702,10 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <comment>Do not localize snupkg and symbols.nupkg</comment>
   </data>
   <data name="Error_RestoreInLockedMode" xml:space="preserve">
-    <value>The packages lock file is inconsistent with the project dependencies so restore can't be run in locked mode. Please disable RestoreLockedMode MSBuild property or pass explicit --force-evaluate flag to run restore to update the lock file.</value>
+    <value>The packages lock file is inconsistent with the project dependencies so restore can't be run in locked mode. Please disable the RestoreLockedMode MSBuild property or pass an explicit --force-evaluate option to run restore to update the lock file.</value>
```

Comment:
Consider rephrasing:

`Unable to restore in locked mode. The packages lock file is inconsistent with the project dependencies. To update the lock file, disable the RestoreLockedMode MSBuild property or use --force-evaluate option.`

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please

Reply:
you're looking at an old commit, I already ~~stopped being polite~~ removed the please.

Reply:
https://github.com/NuGet/Home/issues/8141 :P 

You should go fix it :P